pkg/mc/mc.go

```diff
@@ -161,10 +163,12 @@ func (mc *MC) run(args []string) error {
 
 	output := map[string]json.RawMessage{}
 	for _, c := range contexts {
-		logger.Debug("waiting for next free spot", zap.String("context", c))
-		<-parallelProc
-		logger.Debug("executing", zap.String("context", c))
-		go do(done, c, output, mc.Output == "", mc.Cmd.OutOrStdout(), mc.getKubectlCmd(args, c), mutex)
+		for _, ns := range strings.Split(mc.Namespaces, ",") {
```

Comment:
maybe declare a var namespaces []string variable so we don't have to split multiple times.